src/components/SearchBar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./SearchBar.css";
+
+class SearchBar extends React.Component {
+  state = { searchTerm: "New York" };
```

Comment:
[question] Have you had experience with the new react hooks for state management? Do you have any thoughts on this approach?

Reply:
I have had exposure to the new react hooks for state management, but admittedly not a ton of experience implementing them. Having said that, this definitely seems like a case where we could make use of them, by doing something like `[searchTerm, setSearchTerm] = useState("New York")`, and it seems like we would be able to make that change fairly easily.